tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -109,15 +125,24 @@ public boolean skipStochasticTests() {
   public abstract Processor<T, T> createIdentityProcessor(int bufferSize);
 
   /**
-   * Helper method required for running the Publisher rules against a Publisher.
-   * It must create a Publisher for a stream with exactly the given number of elements.
-   * If {@code elements} is {@code Long.MAX_VALUE} the produced stream must be infinite.
-   *
-   * The stream must not produce the same element twice (in case of an infinite stream this requirement
-   * is relaxed to only apply to the elements that are actually requested during all tests).
-   *
-   * @param elements exact number of elements this publisher should emit,
-   *                 unless equal to {@code Long.MAX_VALUE} in which case the stream should be effectively infinite
+   * Helper method required for creating the Publisher to which the tested Subscriber will be subscribed and tested against.
+   * <p>
+   * By default an <b>asynchronously signalling Publisher</b> is provided, which will use
+   * {@link org.reactivestreams.tck.SubscriberBlackboxVerification#createElement(int)} to generate elements type
+   * your Subscriber is able to consume.
+   * <p>
+   * Sometimes you may want to implement your own custom custom helper Publisher - to validate behaviour of a Subscriber
```

Comment:
In fact, this method should NOT be here! Since we're extending in WithHelperPublisher, also noticed a javadoc `{@link}` error in there, fixed as well.